### ISSUE TITLE
consider consolidating the link / better rendering for keybinding / command in accessibility help dialog

### ISSUE THREAD

**Issue by vscodenpa** [vscodenpa/testissues#43127]
See https://github-com/microsoft/vscode/pull/210116-issue-2236590571

Can we render the markdown such that a screen reader user doesn't have to navigate through [Configure a Keybinding](commandLink)?
